### DISCUSSION TITLE
Sending 410 Gone Header than redirecting to my own error page

### DISCUSSION THREAD

**Discussion by narendra087** [nuxt/nuxt#10170] [Questions (Nuxt 2)]
Hi, I have a website that has content indexed by Google. but I want to permanently delete the content by using http 410 responses and redirecting the page to the error page on my website.

i have tried to use redirect(410, 'page-link'), but the redirect to my error page is not working.

does anyone have a solution? or does anyone have any suggestions how do I permanently delete my content?

**Comment by joroe** [comment]
Have you been able to solve this? I have the same issue.
